Shared/DsaResources.cpp

```diff
@@ -292,6 +292,14 @@ QUrl DsaResources::iconHome() const
   return QStringLiteral("qrc:/Resources/icons/xhdpi/ic_menu_home_dark.png");
 }
 
+/*!
+  \brief Returns the URL to the "KML" icon.
+ */
+QUrl DsaResources::iconKml() const
+{
+  return QStringLiteral("qrc:/Resources/icons/xhdpi/kml.png");
```

Comment:
This should return a QUrl to avoid an implicit cast.

Perhaps this?
```cpp
  return QUrl(QStringLiteral("qrc:/Resources/icons/xhdpi/kml.png"));
```